Makefile

```diff
@@ -48,6 +49,11 @@ $(IMAGE): $(LXC) profile
 # Set up kubectl
 	$(LXC) exec $(INSTANCE) -- snap install kubectl --classic
 
+# Set up postdeploy options
+	$(LXC) exec $(INSTANCE) -- mkdir -p /home/ubuntu/bin
+	$(LXC) exec $(INSTANCE) -- sh -c "echo 'PATH=$$PATH:/home/ubuntu/bin' >> $(USERHOME)/.bashrc"
+	$(LXC) file push ./files/k8s-postdeploy $(INSTANCE)$(USERHOME)/bin/postdeploy
```

Comment:
Is postdeploy intended to detect current system and act accordingly? Or is it specific for k8s? In the first case, should we call the local file postdeploy as well? In the second case, should we instead rename the remote postdeploy to k8s-postdeploy ?

Reply:
I'm fine renaming to k8s-postdeploy